### ISSUE TITLE
[Supporting platform - Make Code - Home Page] After resizing the page upto 400%, vertical and horizontal scroll bar is is not visible on the page.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2377]
**User Experience:**
After zooming the page upto 400%, if the vertical scroll bar does not appears on the page, then the users will find it difficult to perform desired actions on the page.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Navigate through all the controls appearing after select the new product option.
5. Verify the issue.

**Actual Result:**
On zooming the page upto 400%, Vertical and horizontal scroll bar is not visible on the page, due to this user won't be able to access the control available on the page.

**Expected Result:**
On zooming the page upto 400%, Vertical and horizontal scroll bar should be visible on the page, so that user is able to take action on that controls.

**Note:**
Same issue is also exists with Chrome.

**MAS Reference:**
MAS 1.4.10 - Reflow ***

**Labels**
#A11yWCAG2.1, #HCL, #HCL-MakeCode-E2E, #MAS1.4.10, #Win10-Chrome, #Win10-Edge, #accessibility

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

![MAS1 4 10_Resize of 400](https://user-images.githubusercontent.com/30456910/62756573-dcdee780-ba95-11e9-8c51-cdb214282363.jpg)
